hazelcast/serialization/compact.py

```diff
@@ -43,7 +43,11 @@ def __init__(self, compact_serializers: typing.List[CompactSerializer]):
 
     def write(self, out: ObjectDataOutput, obj: typing.Any) -> None:
         clazz = type(obj)
-        serializer = self._type_to_serializer[clazz]
+        serializer = self._type_to_serializer.get(clazz)
+        if serializer is None:
+            raise HazelcastSerializationError(
+                f"No serializer is registered for class/constructor {clazz}."
```

Comment:
```suggestion
                f"No serializer is registered for class {clazz}."
```
There is no constructor for Python

Reply:
Removed at https://github.com/hazelcast/hazelcast-python-client/pull/574/commits/53b1b2f5f916d56afa44207f68d6951697840359.